cluster/openstack-heat/kubernetes-heat/fragments/write-heat-params.yaml

```diff
@@ -0,0 +1,11 @@
+#cloud-config
+merge_how: dict(recurse_array)+list(append)
+write_files:
+  - path: /etc/sysconfig/heat-params
+    owner: "root:root"
+    permissions: "0644"
+    content: |
+      KUBERNETES_SERVER_URL="$KUBERNETES_SERVER_URL"
```

Comment:
`$KUBERNETES_SERVER_URL` is not an environment variable but a heat `str_replace` param name.
Do you think it is better to change heat `str_replace` param names too?